src/centrality/betweenness.jl

```diff
@@ -46,18 +46,17 @@ betweenness: Array{Float64}
 """
 function betweenness_centrality(
     g::SimpleGraph,
-    k::Integer=0;
+    nodes::AbstractArray{Int, 1} = Vector{Int}();
```

Comment:
why do you want a default value instead of a method with arguments `(g,normalize,endpoints)`?

I'm just thinking of avoiding the allocation of the empty array which only gets replaced with `vertices(g)`